### DISCUSSION TITLE
Recommendations for a monastery in Europe with fairly strict environment

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#770] [General]
Someone just asked:

> My friend is looking for recommendations for a monastery in Europe where he could train for a few months. He would like a fairly strict and rigorous environment and has loved training at Zen monasteries in America. Please reply to this thread with ideas if you have them!

## Responses so far

- Thai forest tradition in 
  - CH: https://dhammapala.ch/
  - UK: https://amaravati.org/
- Meditation center (at the hermitage) in UK: https://gaiahouse.co.uk/retreats/personal-retreats/
- https://www.zenrivertemple.org - "I went there, fantastic practice , copy and paste of sojiji temple in Japan  with a rinzai touch as Tekkei roshi is both lineage. His wife is tenzo and has an amazing presence., the cooking samu is very deep.  That's definitely a place to experience."

**Comment by catherinet1** [comment]
The Shaolin Temple Europe? https://www.shaolintemple.eu/index.php?page=kloster-2
'The Shaolin Temple Europe · 歐洲少林寺 is a buddhist monastery in Otterberg / Germany. Studying Chan, Medicine and Training of Kung Fu, Taijiquan, Qi Gong, Rou Gong and more Shaolin Arts belongs to the daily exercise and education of the Shaolin Monks, Masters, Disciples and Novices inside the monastery. The uninterrupted tradition of knowledge which is more than 1,500 years old has been transmitted for generations by Shaolin Masters to their students, constantly evolves and proves in practice to enable people to live a happier and more harmonious life.'
